package.json

```diff
@@ -7,14 +7,14 @@
   "license": "MPL-2.0",
   "scripts": {
     "#postinstall": "cd stumptown && npm install && git checkout package-lock.json",
-    "prettier-check": "prettier --check '**/*.{js,json,scss,html}'",
-    "prettier-format": "prettier --write '**/*.{js,json,scss,html}'",
-    "test": "yarn prettier-check && yarn workspace ssr test && yarn workspace client test",
+    "prettier-check": "prettier --check '**/*.{ts,tsx,json,scss,html}'",
```

Comment:
don't we use `prettier-check` to check the Node code in places like `content/scripts/*`